internal/app/auth-srv.go

```diff
@@ -0,0 +1,211 @@
+package app
```

Comment:
File name suggestion:

`auth-srv.go` -> `server.go`

- This file is already at "appauth" directory, so it's clear that it is about auth.
- Snakecase is not commonly used as file names in Golang.

Reply:
The file is not in the `appauth` folder that because of import cycle. anyway I've re name it to `auth_server`.